RecoVertex/BeamSpotProducer/python/BeamSpotOnline_cff.py

```diff
@@ -3,7 +3,10 @@
 from RecoVertex.BeamSpotProducer.BeamSpotOnline_cfi import *
 
 #scalers = cms.EDProducer('ScalersRawToDigi')
-BeamSpotESProducer = cms.ESProducer("OnlineBeamSpotESProducer")
+BeamSpotESProducer = cms.ESProducer("OnlineBeamSpotESProducer",
+                                    timeThreshold = cms.int32(48),
+                                    sigmaZThreshold = cms.double(2.0)
+                                   )
```

Comment:
```suggestion
import  RecoVertex.BeamSpotProducer.onlineBeamSpotESProducer_cfi as _mod
BeamSpotESProducer = _mod.onlineBeamSpotESProducer.clone()
```
let's reuse already defined default config